build-docs-rsync

```diff
@@ -6,17 +6,33 @@ GIT_RESET="git reset --quiet --hard origin/master"
 cd /local/repos/phpdoc/phd
 $GIT_FETCH && $GIT_RESET >/tmp/log-phd 2>&1
 
-# Update docs sources
-cd /local/repos/phpdoc
-svn up >/tmp/log-svn 2>&1
+BASE_DIR=/local/repos/phpdoc-git
+
+assertUpToDateRepo() {
+    if [ ! -d $BASE_DIR/$1 ]; then
+        git clone https://git.php.net/repository/doc/$2.git $BASE_DIR/$1
+    fi
+    git -C $BASE_DIR/$1 fetch origin && git -C $BASE_DIR/$1 reset --hard origin/master > /tmp/log-git 2>&1
+}
 
-# Build docs
 langs=`php -r 'include "/local/mirrors/phpweb/include/languages.inc"; echo implode( " ", array_keys( $ACTIVE_ONLINE_LANGUAGES ) );'`;
 
+if [ ! -d $BASE_DIR ]; then
+    mkdir $BASE_DIR;
+fi
+
+# Update the core module it self
+assertUpToDateRepo doc-base base
+
+# All translations depend on english being up2date
+assertUpToDateRepo en en
+
+cd /home/docbuilder/phpdoc-git
```

Comment:
Wrong directory, should be `cd $BASE_DIR`.

Reply:
:facepalm: